traced/traced.go

```diff
@@ -323,3 +340,58 @@ Outer:
 		}
 	}
 }
+
+func structMap(obj interface{}) map[string]interface{} {
```

Comment:
can we move this stuff into a different file in the package?

Reply:
Sure.